examples/35-iamidentitymappings.yaml

```
@@ -0,0 +1,30 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-with-iamidentitymappings
+  region: us-east-1
+
+iamIdentityMappings:
+  - arn: arn:aws:iam::000000000000:role/myAdminRole
+    groups:
+      - system:masters
+    username: admin
+    noDuplicateArns: true # prevents shadowing of ARNs
```

Based on the suggestion below:
```suggestion
    noDuplicateARNs: true # prevents shadowing of ARNs
```